app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -214,8 +244,14 @@ class ProfileProgressFragmentTest {
 
   @Test
   fun testProfileProgressFragmentWithProgress_change_configuration_recyclerViewItem0_checkOngoingTopicsCount_countIsTwo() {
-    storyProgressTestHelper.markPartialTopicProgressForFractions(profileId, timestampOlderThanAWeek = false)
-    storyProgressTestHelper.markTwoPartialStoryProgressForRatios(profileId, timestampOlderThanAWeek = false)
+    storyProgressTestHelper.markPartialTopicProgressForFractions(
```

Comment:
If the function name crossing the grey line i.e more than 100 char, you had to put the ktlint disable comment as given in wiki

Reply:
It wasn't having latest develop code I took a pull now. Please check.

Reply:
oh, ok

Reply:
What you can do is just a single line comment, not the block comment. 
[Wiki](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide)

So, this is for when you have a multiple lines of code which all are over 100 char and you cannot break. 
```
/* ktlint-disable max-line-length */
.
.
.
/* ktlint-enable max-line-length */
```

And this one is when we have to disable ktlint check for only one line which is going over 100 char
`// ktlint-disable max-line-length`

Reply:
Done.

Reply:
looks good